src/bin/coverage.rs

```diff
@@ -109,32 +117,27 @@ fn report_file_result(
     }
     // Avoid printing too much, especially not any parse nodes.
     match (result, ambiguity_result) {
-        (Ok(_), Ok(_)) => eprintln!("OK"),
-        (Ok(_), Err(_)) => eprintln!("OK (ambiguous)"),
-        (Err(parse::ParseError::TooShort(handle)), _) => {
+        (Ok(Ok(_)), Ok(_)) => eprintln!("OK"),
+        (Ok(Ok(_)), Err(_)) => eprintln!("OK (ambiguous)"),
+        (Ok(Err(handle)), _) => {
             eprint!("FAIL after ");
 
-            #[cfg(procmacro2_semver_exempt)]
-            {
-                // HACK(eddyb) work around `proc-macro2` `Span` printing limitation
-                let end_location = handle.source_info().end.end();
-                eprintln!("{}:{}", end_location.line, end_location.column);
-            }
```

Comment:
I still think we need this logic though.

Reply:
`ParseError<Span, Range<Span>>` doesn't give us a `Handle` to call `source_info` on, just `{ partial: Option<Range<Span>>, at: Span, expected }`. That's what I've got currently, anyway.

Reply:
The `at`  field is what replaces `handle.source_info().end` - you still need the `.end()` call on that and the rest of the logic, in order to actually get something useful, I think?

Reply:
Ah, yes, that's how to get useful information out of the span.